challenge_1/csharp/fojo/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Fojo
+{
+    public class Challenge1
+    {
+        public static void Main(string[] args)
+        {
+            //Not using linq :)
+            var inputString = Console.ReadLine();
```

Comment:
In the future, I would consider creating a "test suite" where you've hardcoded test cases in and called some `reverseString` function so that you can automate code testing and so that you've sort of documented exactly what test cases you've considered.